src/mscorlib/shared/System/Boolean.cs

```diff
@@ -181,7 +181,7 @@ public int CompareTo(Boolean value)
         public static Boolean Parse(String value)
         {
             if (value == null) throw new ArgumentNullException(nameof(value));
-            return Parse(value.AsReadOnlySpan());
+            return Parse(value.AsSpan());
```

Comment:
I concur.  When I saw this I started digging back to find where the `api-approved` issue was so I could point out that we had said that it should be `AsReadOnlySpan` and we had explictly had issue with `AsSpan`.  Then I saw that this was based off of a review meeting that I must not have been present for, and was marked as approved... and I sighed.

I expect `AsFoo` to give me a `Foo`, not a type sort of like `Foo` but with fewer members.  And especially with variable names like `input` and people who really like `var`, it's misleading to a reviewer, since `input.AsSpan()` may or may not be "more access" than the code needs.

Reply:
Yeah, there are pros and cons here. Naming is hard :-)